docusaurus/docs/test-a-plugin/guides/feature-toggles.md

```diff
@@ -0,0 +1,85 @@
+---
+id: feature-toggles
+title: Feature toggles
+description: How to use feature toggles in e2e tests
```

Comment:
```suggestion
description: How to use feature toggles in E2E tests
```

Reply:
Note that I'm pretty agnostic as to whether e2e or E2E is used, and defer to your judgment. so long as we are consistent. In Google searches, E2E seemed a bit more prominent than e2e in other documentation, so I have suggested a potential change.

Reply:
That's great @josmperez - we should be consistent. I prefer `e2e` over `E2E`.